pkg/model/manifest.go

```diff
@@ -1057,3 +1059,15 @@ func (m *Manifest) IsDeployDefault() bool {
 	}
 	return false
 }
+
+// SetManifestDefaultsFromDev sets context and namespace from the dev
+func (m *Manifest) SetManifestDefaultsFromDev() {
+	if len(m.Dev) == 1 {
+		for _, devInfo := range m.Dev {
+			m.Context = devInfo.Context
+			m.Namespace = devInfo.Namespace
```

Comment:
What would happen if `m.Context` and `m.Namespace` were already set? Is it possible?